tasks/pip.yml

```diff
@@ -58,7 +58,7 @@
 
 - name: Install uwsgi
   pip:
-    name: "{{ item }}"
+    name: http://http.debian.net/debian/pool/main/u/uwsgi/uwsgi_2.0.14+20170111.orig.tar.gz
```

Comment:
I'm not entirely convinced of this bug. Can you create a debian stretch VM with virtualbox or anything you like and test the playbook without this change (leave this line how it was before) and let me know what happens?